diff --git a/setup.py b/setup.py
index fe79684..76ccdd7 100644
--- a/setup.py
+++ b/setup.py
@@ -2,7 +2,8 @@ import glob
 import os
 import platform
 import re
-from pkg_resources import DistributionNotFound, get_distribution, parse_version
+from importlib.metadata import PackageNotFoundError, distribution
+from packaging.version import parse as parse_version
 from setuptools import find_packages, setup
 
 EXT_TYPE = ''
@@ -29,8 +30,8 @@ def choose_requirement(primary, secondary):
     return secondary."""
     try:
         name = re.split(r'[!<>=]', primary)[0]
-        get_distribution(name)
-    except DistributionNotFound:
+        distribution(name)
+    except PackageNotFoundError:
         return secondary
 
     return str(primary)
